salt/states/dockerng.py

```diff
@@ -195,7 +195,8 @@ def _compare(actual, create_kwargs, runtime_kwargs):
                           .format(item, actual_ports))
                 log.trace('dockerng.running ({0}): munged desired value: {1}'
                           .format(item, desired_ports))
-                if actual_ports != desired_ports:
+                difference = set(desired_ports).difference(set(actual_ports))
+                if difference:
                     ret.update({item: {'old': actual_ports,
```

Comment:
What I want is the ports that are in `desired_ports` but not in `actual_ports`.
How do I achieve this with list ?